### ISSUE TITLE
Header icon number text misaligned/insufficient contrast

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#20362]
Cart and wishlist number text is misaligned/bleeds into background at 320px width with text zoomed to 200%, making them unreadable.

<img width="323" height="291" alt="Image" src="https://github.com/user-attachments/assets/703d56e9-2d14-48ac-adb9-b5280977b0c6" />

In Firefox go to Preferences > General > Zoom
- Ensure default Zoom is set to 100%
- check Zoom text only

Create shop with: 
- 1 product
- Wishlist enabled

Visit storefront homepage 
- Set viewport to 320px wide 
- Zoom text to 200% 
- Add product to wishlist
- Add product to cart and close off-canvas
